### DISCUSSION TITLE
How to stop a mesh from rotating?

### DISCUSSION THREAD

**Discussion by alon2138** [enable3d/enable3d#218] [Help / Questions]
I am trying to make a scene with a map as a GLTF mesh and a GLTF animated character, I already have both. Now I need to stop the character from rotating so it looks like the enable3d.io demo, where the character has physics but doesn't rotate so it keeps the balance. Someone knows how to?

**Comment by yandeu** [comment]
I believe you are looking for `body.setLinearFactor` and `body.setAngularFactor`

**Reply by alon2138** [reply]
That was my issue, thanks a lot! Btw I did not think that the creator was going to answer me and even that fast! Just wanted to say that your work is so cool and helpful to start making more advanced games that are playable on the browser.

**Reply by luddensdesir** [reply]
When use setAngularFactor it seems to apply only to future forces being applied, and doesn't do something like brake a rotation. Am I using it wrong? I set it to 50, 50, 50 and the object starts flipping out. I think I misunderstand how it works. I'm simply trying to stop applied angular forces.
